test/SmokeTests/BackendSmokeTests/MSSQL/MSSQL.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.2.0" />
+	<PackageReference Include="Microsoft.DurableTask.SqlServer.AzureFunctions" Version="1.2.2" />
```

Comment:
nit: we should use a wildcard here so that we don't have to update it so frequently. Perhaps even `1.*`?